### PR TITLE
Mu 21

### PR DESCRIPTION
**Pull request by Staroz**
Please, Review this PR.

### CODE REVIEW

File: cypress/support/commands.js [hunk 1]
```diff
@@ -150,6 +150,17 @@ Cypress.Commands.add('addAttachment', (cardName, attachmentLink, linkName) => {
     cy.get('[class="js-add-attachment-url"]').click();
 });
 
+Cypress.Commands.add('copyCard', (cardName, copyCardName, boardName, listName, cardPosition) => { 
+    cy.get('[class="list-card-title js-card-name"]').contains(cardName).click();
+    cy.get('[class="js-sidebar-action-text"]').contains('Copy').click();
+    cy.get('[class="js-autofocus"]').type(copyCardName);
+    cy.get('select.js-select-board').select(boardName+' (current)');
```
Comment (sphinix27) [inline cypress/support/commands.js]: Try to use curly braces for string format.
Example:
```
`${boardName} (current)`
```

File: cypress/support/commands.js [hunk 2]
```diff
@@ -150,6 +150,17 @@ Cypress.Commands.add('addAttachment', (cardName, attachmentLink, linkName) => {
...
+    cy.get('select.js-select-list').select(listName);
+    cy.get('select.js-select-position').select(cardPosition);
+    cy.get('[class="nch-button nch-button--primary wide js-submit"]').click({force: true});
```
Comment (sphinix27) [inline cypress/support/commands.js]: This kind of selectors are not good, try to use only the minimum classes, not all of them, or find something else that can be used to get the element.

**Review by sphinix27 — CHANGES_REQUESTED** [review]
You are missing the update of the fields for the credential.json, please submit that.

**Comment by Staroz** [comment]
Please, review PR please.

**Review by sphinix27 — APPROVED** [review]
LGTM

**Pull request merged**